### PR TITLE
Fix ipc compilation and add ipc feature to test targets

### PR DESCRIPTION
**Pull request by NikVolf**
... to avoid further silent breaks

### CODE REVIEW

File: test.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 # Running Parity Full Test Sute
 
-FEATURES="--features json-tests"
+FEATURES="--features json-tests;ipc"
```
Comment (tomusdrw) [inline test.sh]: AFAIR it should be comma-separated, is it working with semicolon too?

Comment (NikVolf) [inline test.sh]: no, semicolon, not working with comma

Comment (tomusdrw) [inline test.sh]: Seems it might have been changed in beta, `stable` is complaining about missing feature on Travis.

Comment (NikVolf) [inline test.sh]: ok using that
```
cargo run --features json-tests ipc
```

File: test.sh [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 # Running Parity Full Test Sute
 
-FEATURES="--features json-tests"
+FEATURES="--features json-tests ipc"
```
Comment (tomusdrw) [inline test.sh]: Maybe try `"--features \"json-tests ipc\""`

Comment (NikVolf) [inline test.sh]: what for? `test.sh` is running as it is now

Comment (tomusdrw) [inline test.sh]: I'm pretty sure it won't work:
`cargo test test_name` - runs only tests with `test_name`
`cargo test --features json-tests ipc` - runs only tests with `ipc`

Comment (NikVolf) [inline test.sh]: ah no, this is actually becomes test argument

**Comment by NikVolf** [comment]
@tomusdrw should be ok now

**Pull request merged**